updater/reports/Report.py

```diff
@@ -80,19 +80,28 @@ def scriptPath(self, script):
 	# Executes a script but prints stderr and returns stdout only
 	def executeScript(self, script, stdin = None):
 		if self.configuration["remoteRun"]["enabled"]:
-			try:
-				with open(script) as f:
-					assert(stdin == None)
-					stdin = f.read()
-				command = ["bash -s", "--"]
-			except:
-				command = script
+			if not stdin:
+				# A script is either a path to a file or a list of strings
```

Comment:
Maybe we should put this explanation in the comment above `executeScript` so that this is immediately clear when reading the function? And I wouldn’t mind having this information here as well.